packages/store-indexer/bin/sqlite-indexer.ts

```diff
@@ -30,24 +33,36 @@ const env = z
     }),
   });
 
-const chain = possibleChains.find((c) => c.id === env.CHAIN_ID);
-if (!chain) {
-  throw new Error(`Chain ${env.CHAIN_ID} not found`);
-}
+const chain = env.CHAIN_ID ? possibleChains.find((c) => c.id === env.CHAIN_ID) : undefined;
+
+const transports: Transport[] = [
+  env.RPC_WS_URL ? webSocket(env.RPC_WS_URL) : null,
+  env.RPC_HTTP_URL ? http(env.RPC_HTTP_URL) : null,
+].filter(isNotNull);
```

Comment:
should we throw an error if neither `CHAIN_ID` nor `RPC_HTTP_URL`/`RPC_WS_URL` are set?